edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -136,7 +136,7 @@ var edx = edx || {};
         updateRemainingTime: function (self) {
             self.timerTick ++;
             self.secondsLeft --;
-            if (self.timerTick % 30 === 0){
+            if (self.timerTick % 60 === 0){
```

Comment:
I also noticed that this line doesn't match our ESLint rules, so I guess that isn't applied in this repo! There should be a space before the opening ``{``.